provider/resource_neap.go

```diff
@@ -154,7 +154,7 @@ type neapModel struct {
 }
 
 func (n neapModel) objectype() string {
-	return itsiResourceTypeNEAP
```

Comment:
if you don't want to add itsi as prefix, that's fine, probably we need to add resource name to a metadata then :)